### DISCUSSION TITLE
[PSD1] Exercise 1.16

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#112] [Esercizi (A-L)]
**Exercise 1.16** Convert the following unsigned binary numbers to exadecimal. Show your work.

(a) $1110_2$
(b) $100100_2$
(c) $11010111_2$
(d) $011101010100100_2$

- _**N.B.:** I numeri da convertire sono gli stessi dell'Esercizio 1.14 (#111 ) ma questa volta sono da convertire in esadecimale_

**Comment by niccolo-fato** [comment]
![photo_5870846455839833119_y](https://user-images.githubusercontent.com/73995334/210087061-8e654ab2-cc67-4c44-9818-4fdd303ee37a.jpg)
